Source/CNTK/BrainScript/CNTKCoreLib/CNTK.core.bs

```diff
@@ -82,7 +82,7 @@ ConvolutionalLayer {numOutputChannels,   # e.g. (1) or BS.Constants.None
                     initValueScale = 1,          # TODO: rename to initScale
                     initBias = 0,
                     #reductionRank = 1,          # TODO: support this
-                    stride = 1, pad = false,
+                    stride = 1, dilation=1, pad = false,
```

Comment:
Please make any necessary change to keep compatibility, it could be easier on your side with more test available.
This code has been in side branch for a long time and I could have introduced some compatibility issue merging upstream change over time.